aspnetcore/security/authentication/scaffold-identity/consoleAddUser/Data/SeedData.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Threading.Tasks;
+using WebApplication1.Data;
+
+namespace ContactManager.Data
+{
+    #region snippet
+    public static class SeedData
+    {
+        public static async Task Initialize(IServiceProvider serviceProvider, string userList)
+        {
+            using (var context = new ApplicationDbContext(
+                serviceProvider.GetRequiredService<DbContextOptions<ApplicationDbContext>>()))
+            {
+                var userPW = GetNextUserGeneratePW(userList);
```

Comment:
@BillWagner how can I move `GetNextUserGeneratePW(userList)` inside the loop, so I don't need to call it in two places?

Here's the code
```
public static async Task Initialize(IServiceProvider serviceProvider, string userList)
{
    using (var context = new ApplicationDbContext(
        serviceProvider.GetRequiredService<DbContextOptions<ApplicationDbContext>>()))
    {
        var userPW = GetNextUserGeneratePW(userList);   // MOVE INSIDE LOOP
        while (userPW.user != null)
        {
            var userID = await EnsureUser(serviceProvider, userPW.password, userPW.user);
            NotifyUser(userPW);
            userPW = GetNextUserGeneratePW(userList);   // FIX ME
        }
    }
}
```